.github/workflows/pr.yml

```diff
@@ -0,0 +1,13 @@
+name: pr
+on: pull_request_target
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: auto-merge
+      id: auto-merge
+      run: gh pr merge --auto "$PR_URL"
```

Comment:
You might want to add `--rebase` to your list of run flags to rebase.

Reply:
I have fixed this in a different way.